src/python/WMCore/MicroService/DataStructs/MSRuleCleanerWflow.py

```diff
@@ -120,8 +122,8 @@ def _wfParse(self):
                     self.extDoc[keyName] = {}
                     for item in data['values']:
                         self.extDoc[keyName].update(item)
-                elif (isinstance(data['type'], tuple) and (str in data['type'] or unicode in data['type'])) or \
-                     (data['type'] is str or data['type'] is unicode):
+                elif (isinstance(data['type'], tuple) and (bytes in data['type'] or str in data['type'])) or \
```

Comment:
rhetorical comment, what a horrible line!

Reply:
I also lit up a candle when I pushed this, praying it does not break. Oh, let me create an issue to track this, we should definitely explicitely check that this work in py3.

Reply:
Created #10323 to track this

Reply:
A rhetorical reply. Thank you @amaltaro